### DISCUSSION TITLE
Compile with AAC Support

### DISCUSSION THREAD

**Discussion by MadTiXx** [fayaaz/mixxx-pi-gen#21] [Ideas]
Hi,
is it possible to compile next build with AAC Support? 
https://github.com/mixxxdj/mixxx/wiki/Compiling-On-Linux#user-content-optional-build-with-m4aaac-file-support

Greetings

**Comment by fayaaz** [comment]
I messaged them about this here: https://mixxx.zulipchat.com/#narrow/stream/247620-development-help/topic/Building.20in.20Linux.20with.20AAC.20support

We are building on debian so there may be a problem with offsets if you're importing a previously scanned library. I will add that to the build options and let you know when there's an image you can test it out on.

**Reply by fayaaz** [reply]
@MadTiXx  https://github.com/fayaaz/mixxx-pi-gen/suites/2640928804/artifacts/57985097 could you try this image (it may be zipped up twice just fyi)

**Comment by MadTiXx** [comment]
@fayaaz Thanks! Works perfect with my m4a-Files :-)
